recipes/pyramid_log/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pyramid_log" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 165ce9460548b09061db6023b5079eb6290d9490a2b4cf28946e3515185d0a52
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```

Comment:
Please remove the `build` section - when not using compilers, you put the dependencies in `host`.

Reply:
I was wondering what the difference was.

Reply:
You can find better explanation here, https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#requirements-section.